upgrade/upgrade-scripts/execute

```diff
@@ -17,6 +17,8 @@
 
 . "${BASH_SOURCE%/*}/common.sh"
 
+INTERFACE_TO_MACADRESS_MAP_FILE_PATH="/etc/interface_to_macaddress_map.out"
```

Comment:
I don't feel strongly, but I think maybe `/var/lib/delphix-upgrade` or `/var/tmp/delphix-upgrade` or something might be better than putting this file in `/etc`.. maybe not a big deal, but this file isn't really a configuration file, but more like a file containing state needed for the `systemd-networkd` service..

Reply:
Yes, thanks for suggesting this. I was not aware we had a /var/tmp/delphix-upgrade directory and it seems apt for this purpose.